pymc3/distributions/dist_math.py

```diff
@@ -59,6 +59,7 @@ def bound(logp, *conditions, **kwargs):
     logp: float
     *conditions: booleans
     broadcast_conditions: bool (optional, default=True)
+    model: Model (optional, default=None)
```

Comment:
Is the `model` kwarg used anywhere? If not, I would remove it altogether. If it crops up somewhere it's probably in the tests, and that's not a good enough reason to add an extra keyword.